bottles/backend/managers/manager.py

```diff
@@ -738,6 +740,28 @@ def process_bottle(bottle):
             if conf_file_yaml.get("run_in_terminal"):
                 conf_file_yaml["run_in_terminal"] = False
 
+            # Check if the path in the bottle config corresponds to the folder name
+            # if not, change the config to reflect the folder name
+            # if the folder name is "illegal" accross all platforms, rename the folder
+            sain_name = pathvalidate.sanitize_filepath(_name, platform='universal') # "universal" platform works for all filesystem/OSes
```

Comment:
What does sain mean? I imagine it's a typo, perhaps you meant `sane_name`?

Reply:
`sain` is french for `sane`, I was probably tired when I wrote this